src/models/Step.d.ts

```diff
@@ -7,7 +7,7 @@ declare class StepSpec extends Spec {
 }
 
 declare class Step {
-  spec(name?: string, data?: any): StepSpec;
+  spec<T = any>(name?: string, data?: T): StepSpec;
   clean(name?: string, data?: any): CleanStep;
```

Comment:
could you please add the same generic support for clean steps

Reply:
Done